File: app/src/main/java/br/com/flasco/leanworks/utils/PresentationUtils.java
package br.com.flasco.leanworks.utils;

import android.content.Intent;

public class PresentationUtils {

    public static Intent shareIntent(String content){
        Intent sendIntent = new Intent();
        sendIntent.setAction(Intent.ACTION_SEND);
        sendIntent.putExtra(Intent.EXTRA_TEXT, content);
        sendIntent.setType("text/plain");

        Intent shareIntent = Intent.createChooser(sendIntent, null);
        return shareIntent;
    }
}

File: app/src/main/java/br/com/diogomoreno/events/utils/PresentationUtils.kt
package br.com.diogomoreno.events.utils

import android.content.Intent

object PresentationUtils {
    fun shareIntent(content: String?): Intent {
        val sendIntent = Intent()
        sendIntent.action = Intent.ACTION_SEND
        sendIntent.putExtra(Intent.EXTRA_TEXT, content)
        sendIntent.type = "text/plain"
        return Intent.createChooser(sendIntent, null)
    }
}